frontend/src/composables/useEscape.js

```diff
@@ -0,0 +1,17 @@
+import { onMounted, onUnmounted } from "vue";
```

Comment:
Is that necessary if we can unify on `Dialog.vue`?

Reply:
No, it would not be necessary.